### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by IjayAbby**
In this project I was required to fulfil the following;

- Add a book ie, Title and Author
- Delete a book
- Update the book in Local Storage.

### CODE REVIEW

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @IjayAbby , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️

- [x] Your project is working absolutely fine. But as per project requirements, we have to use the function rather than using the classes. Kindly try to achieve the functionality of adding and removing the books using functions. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Try to use array methods that don't mutate the array, e.g. concat, filter, slice instead of push, pop, splice and so on. 

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      || e.name === 'QuotaExceededError'
+      // Firefox
+      || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+      // acknowledge QuotaExceededError only if there's something already stored
+      && (storage && storage.length !== 0);
+  }
+}
+
+function setItem(name, item) {
+  if (localStorageAvailable()) {
```
Comment (MahmoudBakr23) [inline app.js:27]: Great work using functions to build the app 👏🏻

File: app.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  saveAndDisplay();
+}
+/* eslint-enable no-use-before-define */
+
+const addBtn = document.querySelector('#addBtn');
+addBtn.addEventListener('click', () => {
+  const inputTitle = document.querySelector('.inputTitle');
+  const inputAuthor = document.querySelector('.inputAuthor');
+
+  addBook(inputTitle.value, inputAuthor.value);
```
Comment (MahmoudBakr23) [inline app.js:110]: ### **Optional**
It'd be better if you used the `. reset()` method after the submission to empty the inputs for new entries. 😊🙏🏻

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @IjayAbby 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**